src/assets/stylesheets/_home.scss

```diff
@@ -225,31 +474,6 @@ header.primary-header {
   }
 }
 
-.aptible-tools {
-  .tools-carousel {
-    margin: 70px auto 0;
-    width: 330px;
-  }
-
-  .slick-slide {
-    img {
-      margin: 0 auto;
-      max-width: 100%;
-
-      &.django {
-        width: 180px;
-        margin-top: 25px;
-      }
-
-    }
-  }
-
-  .tools-description {
-    text-align: center;
-    margin: 40px 0 0;
-  }
-}
-
```

Comment:
Could clean up more styles here. Tabling for the next big round that may included ditching Assemble for Middleman.